lib/active_model/serializer/adapter/json.rb

```diff
@@ -2,10 +2,15 @@ class ActiveModel::Serializer::Adapter::Json < ActiveModel::Serializer::Adapter
         extend ActiveSupport::Autoload
         autoload :FragmentCache
 
+        def initialize(serializer, options = {})
+          super
+          @included = ActiveModel::Serializer::Utils.include_args_to_hash(@options[:include] || '*')
+        end
+
         def serializable_hash(options = nil)
           options ||= {}
           if serializer.respond_to?(:each)
-            result = serializer.map { |s| FlattenJson.new(s).serializable_hash(options) }
+            result = serializer.map { |s| FlattenJson.new(s, include: @included).serializable_hash(options) }
```

Comment:
As for the fallback, I'm not too sure about delegating it to `include_args_to_hash` because we might want a different fallback depending on the adapter (`'', '*', '**').